src/components/SettingsForm.vue

```diff
@@ -100,18 +100,43 @@
 				type="text"
 				placeholder="quota">
 		</p>
+		<p>
+			<label for="mapping-quota">{{ t('user_oidc', 'Groups mapping') }}</label>
+			<input id="mapping-quota"
+				v-model="localProvider.settings.mappingGroups"
+				type="text"
+				placeholder="groups">
```

Comment:
This input's state could be disabled if "Use group provisioning" is not checked.